aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -1254,6 +1312,24 @@ module aptos_token::token {
         token_data.uri
     }
 
+    public fun get_tokendata_description(creator: address, token_data_id: TokenDataId): String acquires Collections {
```

Comment:
We don't need `creator`, which can get from token_data_id.creator.  Similar to other get functions. Unfortunately, cannot change `get_tokendata_uri` now

Reply:
great point, just updated : )